tests/posix/common/src/pthread.c

```diff
@@ -405,6 +405,28 @@ ZTEST(pthread, test_sched_getscheduler)
 
 	zassert_true((rc == -1 && err == ENOSYS));
 }
+ZTEST(pthread, test_sched_setparam)
+{
+	struct sched_param param;
+
+	param.sched_priority = 2;
```

Comment:
can probably use static initialization

```suggestion
	struct sched_param param = {
		.sched_priority = 2,
	};
```